extensions-contrib/tdigestsketch/src/main/java/org/apache/druid/query/aggregation/tdigestsketch/TDigestSketchAggregator.java

```diff
@@ -56,16 +58,24 @@ public TDigestSketchAggregator(ColumnValueSelector selector, @Nullable Integer c
   @Override
   public void aggregate()
   {
-    if (selector.getObject() instanceof Number) {
+    Object obj = selector.getObject();
+    if (obj == null) {
+      return;
+    }
+    if (obj instanceof Number) {
       synchronized (this) {
-        histogram.add(((Number) selector.getObject()).doubleValue());
+        histogram.add(((Number) obj).doubleValue());
       }
-    } else if (selector.getObject() instanceof MergingDigest) {
+    } else if (obj instanceof MergingDigest) {
       synchronized (this) {
-        histogram.add((MergingDigest) selector.getObject());
+        histogram.add((MergingDigest) obj);
       }
     } else {
-      TDigestSketchUtils.throwExceptionForWrongType(selector);
+      throw new IAE(StringUtils.format(
```

Comment:
`StringUtils.format` is not necessary when using `IAE` as it already accepts format arguments